src/SFA.DAS.ApprenticeFeedback.Domain/Entities/ApprenticeFeedbackTarget.cs

```diff
@@ -107,32 +107,14 @@ public void UpdateApprenticeshipFeedbackTarget(Learner learner, ApplicationSetti
 
             Withdrawn = apprenticeStatus == ApprenticeshipStatus.Stopped;
 
-            DateTime? GetApprenticeStoppedDate(Learner learner)
-            {
-                if (!learner.ApprovalsStopDate.HasValue && !learner.LearnActEndDate.HasValue)
-                {
-                    return null;
-                }
-                else if (learner.ApprovalsStopDate == null)
-                {
-                    return learner.LearnActEndDate;
-                }
-                else if (learner.LearnActEndDate == null)
-                {
-                    return learner.ApprovalsStopDate;
-                }
-                else
+            EndDate = learner.LearnActEndDate 
+                ?? apprenticeStatus switch
                 {
-                    // Return the earliest date if both set.
-                    return learner.LearnActEndDate > learner.ApprovalsStopDate ? learner.ApprovalsStopDate : learner.LearnActEndDate;
-                }
-            };
-            EndDate = apprenticeStatus switch
-            {
-                ApprenticeshipStatus.Stopped => GetApprenticeStoppedDate(learner),
-                ApprenticeshipStatus.Paused => learner.ApprovalsPauseDate,
-                _ => learner.EstimatedEndDate,
-            };
+                    ApprenticeshipStatus.Stopped => learner.ApprovalsStopDate,
+                    ApprenticeshipStatus.Paused => learner.ApprovalsPauseDate,
+                    _ => learner.EstimatedEndDate,
+                } 
+                ?? learner.EstimatedEndDate;
```

Comment:
Is line 117 intentional? It doesn't exist on master

Reply:
It is - that would be AC3 & AC6